packages/SwingSet/src/kernel/state/vatKeeper.js

```diff
@@ -240,26 +243,28 @@ export function makeVatKeeper(
    * @param {string} msg  The message to append.
    */
   function addToTranscript(msg) {
-    const id = Nat(Number(storage.get(`${vatID}.t.nextID`)));
-    storage.set(`${vatID}.t.nextID`, `${id + 1}`);
-    storage.set(`${vatID}.t.${id}`, JSON.stringify(msg));
+    const id = advance(`${vatID}.t.nextID`);
+    storage.set(`${vatID}.t.${id}`, JSONstringify(msg));
   }
 
   function vatStats() {
-    function getStringAsNat(ostr) {
-      return Nat(Number(storage.get(ostr)));
+    function compareToFirst(key, first) {
```

Comment:
The meaningful idea here is not `compareToFirst`, it's `subtract`.  We actually want the difference value itself, not just the sign (even though this provides that, the name is confusing).

Reply:
Oh sure, I had forgotten that the name might imply the wrong thing